src/ui/theme/fonts/icons.scss

```diff
@@ -57,3 +57,9 @@
 .igui_icon_play:before {
   content: "\e90a";
 }
+.igui_icon_rewind:before {
+  content: "⏪";
```

Comment:
I'll dig up the webicon font and provide you with two material icons for rewind & forward.
When the feature is in a separate PR, I can easily update the webfont there.